.env

```diff
@@ -0,0 +1,117 @@
+# This file only contains Docker relevant variables.
```

Comment:
hopefully we don't need anything in this file?

Reply:
I'll be working on cleaning this up today and see where we get

Reply:
got it. the version that we use now should probably be the protocol version, instead of the airbyte version, i think? not that the env variable has to change. just that the semantic meaning of it is different.

other than version, are we using anything else from this file? i would hope not since there's no main method in this module so there shouldn't be any environment to set.

Reply:
yes, I agree we should use different versions from the airbyte version, and we can get rid of this file if we just set the version info elsewhere (like in gradle.properties for instance).

Reply:
So basically i removed everything but version in this file. We could set it in gradle.properties if we want, but right now it's the only thing in this file.